.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
```

Comment:
create `.env.sample` file and add only keys which has been used in `.env` file.
for eg.
```
DB_HOST=
DB_USERNAME=
DB_PORT=
DB_DATABASE=
DB_PASSWORD= 
```

Reply:
done.